io.openems.edge.ess.kaco.blueplanet.gridsave50/src/io/openems/edge/ess/kaco/blueplanet/gridsave50/EssKacoBlueplanetGridsave50.java

```diff
@@ -47,6 +47,7 @@
 import io.openems.edge.common.event.EdgeEventConstants;
```

Comment:
I added it as a TODO in the code. For now I want to keep it, so that we can test the GenericEss + Battery-Inverter implementation carefully on few systems, before migrating everything.